### DISCUSSION TITLE
귀 슬라임 지문 참고용

### DISCUSSION THREAD

**Discussion by cyannn1** [uotalkie/dol-kr#160] [General]
https://docs.google.com/spreadsheets/d/1d_cuAtF9JAvCD6Bw3HOEY9UEsbJz10Hz6he9wdqxSEg/edit?gid=1057271415#gid=1057271415

시급한 일은 아니지만, 참고용으로 정리해서 올려둡니다...(빠뜨린 번역 예시가 있을 수도 있어요.)

**Comment by mansus-traveler** [comment]
case 추가해주시면 감사하겠습니다.
1) `You know the ear slimes wouldn't let you ~A~ without ~B~` : 당신은 귀 슬라임이 `~B~`하지 않고는 `~A~` 못하게 하리라는 것을 안다.

예시: `You know the ear slimes wouldn't let you leave without flaunting your chest.` → 당신은 귀 슬라임이 당신의 가슴을 과시하지 않고는 떠나지 못하게 하리라는 것을 안다.
예시2: `You know the ear slimes wouldn't let you do anything but expose yourself.` → 당신은 귀의 슬라임이 당신을 노출시키지 않고는 아무것도 못하게 하리라는 것을 안다.

**Reply by cyannn1** [reply]
체크해주셔서 감사합니다! 반영했습니다.
